### DISCUSSION TITLE
Is there a way to modify response body in fairing?

### DISCUSSION THREAD

**Discussion by 3olkin** [rwf2/Rocket#1816] [Questions]
I'm writing JSON api with authorization and I have to refresh jwt tokens on each request and sent them back with response, so I'd like to have a fairing which will do it. Or should this part of application logic be handled by Responder?

**Comment by knota-lab** [comment]
https://rocket.rs/v0.5-rc/guide/fairings/#fairings

![image](https://user-images.githubusercontent.com/26835087/129434973-49817f2d-f72a-4b0c-86f2-56026c58dcaf.png)
